tests/test_transpiler.py

```
@@ -106,3 +106,129 @@ def test_pipeline_operator_with_args():
     clyp_code = 'data |> clean |> transform("fast") |> save'
     parsed_code = parse_clyp(clyp_code)
     assert 'save(transform(clean(data), "fast"))' in parsed_code
+
+import tempfile
```

Ruff (0.11.9)

`tempfile` imported but unused

Remove unused import: `tempfile`

(F401)